File: src/main/java/client/database/data/SkillData.java
package client.database.data;

public class SkillData {
   private int skillId;

   private byte skillLevel;

   private int masterLevel;

   private long expiration;

   public SkillData(int skillId, byte skillLevel, int masterLevel, long expiration) {
      this.skillId = skillId;
      this.skillLevel = skillLevel;
      this.masterLevel = masterLevel;
      this.expiration = expiration;
   }

   public int getSkillId() {
      return skillId;
   }

   public byte getSkillLevel() {
      return skillLevel;
   }

   public int getMasterLevel() {
      return masterLevel;
   }

   public long getExpiration() {
      return expiration;
   }
}

File: engine/model/src/main/scala/client/database/data/SkillData.scala
package client.database.data

class SkillData(private var _skillId: Int, private var _skillLevel: Byte, private var _masterLevel: Int,
                private var _expiration: Long) {
  def skillId: Int = _skillId

  def skillLevel: Byte = _skillLevel

  def masterLevel: Int = _masterLevel

  def expiration: Long = _expiration
}
